static-site/posts/updates/2019.3.4-update.udon

```diff
@@ -0,0 +1,54 @@
+:-  :~  date/'~2019.3.14'
+        title/'~2019.3.4 Update'
+        preview/'A short update from the Tlon team for the week of ~2019.3.4.'
+        author/'~hidrel-fabtel'
+        type/'post'
+    ==
+;>
+
+# ~2019.3.4 Update
+
+Here’s a short update from the team for the week of `~2019.3.4`:
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.3.4-update-1.jpg"(width "100%");
+
+## Network
+
+We breached successfully. You can read about [what we fixed](https://github.com/urbit/arvo/pull/1101). Issues ocassionally crop up as people join the network, but it looks to be smooth sailing as we make our way toward [cc-release](https://github.com/urbit/urbit/tree/cc-release) and new ames.
+
+## @p Bug
+
+The nice folks behind [@urbitlive](https://twitter.com/urbitlive) discovered a bug in @p leading to planet name-collisions. Due to a subtly incorrect implementation of the cryptographic algorithm we use to scramble planet names – used to dissociate them from their parent stars – a small percentage of people with different Azimuth points could end up with the same planet name. For example: 
+
+```
+> `@p`3.108.299.008
+~morlyd-mogmev
+> `@p`479.733.505
+~morlyd-mogmev
+```
+
+We [announced the bug on Twitter](https://twitter.com/urbit/status/1105850450559918082) today, and [a conversation](https://github.com/urbit/arvo/issues/1105) is ongoing on Github.
+
+A number of solutions have been proposed – from fixing the algorithm and using a constraint solver to find a new ordering of phonemes such that all issued planets would keep their current names to hard-coding a new name for every affected planet – but none nearly as elegant as [this epic 3-line suggestion](https://github.com/urbit/arvo/issues/1105#issuecomment-472585937) from @max19, which simply shifts the affected planet names into unused parts of the available name space.
+
+Stay tuned for a post with a full report.
+
+## Team
+
+*Jared* `~nidsut-tomdun` spent the week on the @p debacle and more than once generated all four billion planet names to check for collisions. *Ted* `~rovnys-ricfer` spent much of the week thinking about redesigning the PKI only to realize that the existing design is quite good. *Logan* `~tacryt-socryp` and *Isaac* `~novlud-padtyv` started working on the next major revision to Landscape. *Ben* `~pitmug-roptic` quietly [started working on](https://github.com/urbit/urbit/pull/1221) 64-bit Vere. Harison `~pilfer-pandex` [brought the hammer down](https://github.com/urbit/docs/pull/399#issuecomment-472667240) on the pronunciation debate.
```

Comment:
Assuming @pilfer-pandex is okay with their name getting leaked, you could at least spell it right. (;